e2e/pod.go

```diff
@@ -395,3 +395,22 @@ func getKernelVersionFromDaemonset(f *framework.Framework, ns, dsn, cn string) (
 
 	return kernelRelease, nil
 }
+
+func recreateCSIPods(f *framework.Framework, podLabel, daemonsetName, deploymentName string) error {
+	err := deletePodWithLabel(podLabel,
+		cephCSINamespace, false)
+	if err != nil {
+		return fmt.Errorf("failed to delete pods with labels: %w", err)
```

Comment:
`with labels`? which labels? it is good to include those details in the error.

Reply:
Done